src/main/resources/admin/tools/content-type-manager/components/form-definition/component-form-definition-x-datas.html

```diff
@@ -11,6 +11,7 @@
                   v-model="xDatas[index].attributes.name"
                   :label="fields.label"
                   :hint="fields.hint"
+                  :rules="[rules.requiredText, rules.noSpaces]"
```

Comment:
Umm i cant read from the docs that x-data is allowed on a content type. 
@alansemenov is this a feature i don't know about? 
It should be added to the site.xml.

Reply:
@poi33 Yes, x-data can be used in a content type schema. [Here](https://github.com/enonic/app-features/blob/master/src/main/resources/site/content-types/using-mixins/using-mixins.xml)'s example from app-features. And [here](https://developer.enonic.com/docs/xp/stable/cms/content-types#extend_any_content_type)'s very sparse docs which should be extended with a sample of x-data used inside content type xml.